code/States/StateHandler.cs

```diff
@@ -35,6 +35,7 @@ public void ChangeState( BaseState state )
 		[Event.Tick.Server]
 		private void Tick()
 		{
+			Game.Instance.State = State;
```

Comment:
You shouldn't need to do this anymore as the game state networks properly.

Reply:
fair

Reply:
Should be fixed now